Modules/Loadable/VolumeRendering/Logic/vtkSlicerVolumeRenderingLogic.cxx

```diff
@@ -948,6 +948,18 @@ void vtkSlicerVolumeRenderingLogic::UpdateDisplayNodeFromVolumeNode(
 }
 
 //----------------------------------------------------------------------------
+void vtkSlicerVolumeRenderingLogic
+::UpdateDisplayNodeFromVolumeNode(vtkMRMLVolumeRenderingDisplayNode *paramNode,
+                                  vtkMRMLVolumeNode *volumeNode)
+{
+  vtkMRMLVolumePropertyNode *propNode =
+    paramNode ? paramNode->GetVolumePropertyNode() : NULL;
```

Comment:
It looks that it actually was the default behavior: when calling this function, it would just pass a NULL vtkMRMLVolumePropertyNode and a NULL vtkMRMLAnnotationROINode as 3rd and 4th args to the same function (UpdateDisplayNodeFromVolumeNode) with 4 args.

I guess we could trigger a warning if they're missing from the input vtkMRMLVolumeRenderingDisplayNode. I'm not 100% sure of how this code is expected to behave since I did not write that commit, but I'll adapt to whichever once we figure it out.